### PR TITLE
Fixed white blocks after shuffling + allowed reduced motion

### PR DESCRIPTION
**Pull request by BudzioT**
Some weird white blocks were showing up after shuffling at the top and bottom, fixed that.
Also made animation's duration 0 when prefers-reduced-motion is set

### CODE REVIEW

**Review by Gitstar-OC — APPROVED** [review]
👍

File: src/app/harbor/battles/battles.tsx [hunk 1]
```diff
@@ -166,7 +166,10 @@ export default function Matchups({ session }: { session: HsSession }) {
       if (currentStep >= totalSteps) {
         clearInterval(spinInterval)
         document.body.style.transform = 'rotate(0deg)' // reset to initial state
-        document.body.style.paddingTop = '1px'
+        document.body.style.paddingTop = '0'
```
Comment (Gitstar-OC) [inline src/app/harbor/battles/battles.tsx:169]: Just a question deleting this line may work why we are setting it to 0  ?

Comment (Gitstar-OC) [reply src/app/harbor/battles/battles.tsx:169]: Same for the others.

Comment (BudzioT) [reply src/app/harbor/battles/battles.tsx:169]: Actually we don't need to set it to 0, I was just testing stuff and left it - transform and willChange is enough

Comment (BudzioT) [reply src/app/harbor/battles/battles.tsx:169]: But it doesn't change anything, as it's initially 0 too, so Ig it's up to reviewer to leave it or not ; )

Comment (malted) [reply src/app/harbor/battles/battles.tsx:169]: Setting it to 1px fixed the white blocks issue before which was why it was included FYI

**Comment by malted** [comment]
Tested - LGTM!

**Pull request merged**